thresher/models.py

```diff
@@ -135,18 +155,21 @@ class Answer(models.Model):
     answer_id = models.IntegerField()
 
     # The question to which this answer belongs
-    question = models.ForeignKey(Question, related_name="answers")
+    question_id = models.OneToOneField(QuestionContent)
     
     # The text of the amswer
-    text = models.TextField()
+    answer_content = models.TextField()
+
+    # The next question the answer is leading to
+    next_question_id = models.OneToOneField(QuestionContent)
```

Comment:
I think this should also be a a ForeignKey since, similar to above, the "next question" could be referred to from multiple answers.